### PR TITLE
19034 Rebasing legal name feature branch

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#19034

*Description of changes:*
Rebased the legal name feature branch with main

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-637-v17badd4.web.app

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "business-create-ui",
-  "version": "5.6.0",
+  "version": "5.9.0",
```
Comment (severinbeauvais) [inline package.json:3]: I'd change this to 6.0.0 as it's a major change.

**Comment by severinbeauvais** [comment]
That's too many files to review. Are these just the commits from main branch since they diverged? If so then I propose that they don't need to be reviewed.

On the other hand, if there were merge conflicts then some files should be reviewed. LMK

**Comment by JazzarKarim** [comment]
> That's too many files to review. Are these just the commits from main branch since they diverged? If so then I propose that they don't need to be reviewed.
> 
> On the other hand, if there were merge conflicts then some files should be reviewed. LMK

Yes, most of them are just commits that came from the main branch. No need to review any of those.

Sev, the files that I think can be reviewed (major merge conflicts) are:
- App.vue
- BreadcrumbResource.ts
- EntityInfo.vue
- business-interface.ts
- FetchConfig.ts

Other than that, it was small stuff conflicts that I'm pretty sure it should be OK.

File: src/interfaces/store-interfaces/state-interfaces/business-interface.ts
Comment (severinbeauvais) [inline src/interfaces/store-interfaces/state-interfaces/business-interface.ts:1]: Please compare this file with what's in main branch. The new file below has 53 lines but the one in main has only 43.

**Comment by severinbeauvais** [comment]
I have reviewed the following files and I trust the other ones to be OK.

- [x] App.vue
- [x] BreadcrumbResource.ts
- [x] EntityInfo.vue
- [x] business-interface.ts
- [ ] ~FetchConfig.ts~ not in diff; did you mean "src/utils/feature-flag-utils.ts"?

File: src/interfaces/store-interfaces/state-interfaces/business-interface.ts
Comment (JazzarKarim) [reply src/interfaces/store-interfaces/state-interfaces/business-interface.ts:1]: Yes Sev! There's a new interface which is probably why the difference:
business-interface

This was from your PR a while back.

**Comment by JazzarKarim** [comment]
> I have reviewed the following files and I trust the other ones to be OK.
> 
> * [x]  App.vue
> * [x]  BreadcrumbResource.ts
> * [x]  EntityInfo.vue
> * [x]  business-interface.ts
> * [ ]  ~FetchConfig.ts~ not in diff; did you mean "src/utils/feature-flag-utils.ts"?

Thank you Sev! I think the feature flag utils file is OK. No conflicts there. Never mind then the `FetchConfig.ts` file.

**Comment by severinbeauvais** [comment]
@JazzarKarim What's the game plan for new PRs on main branch? Do they need to be duplicated for the legal name branch?

**Comment by JazzarKarim** [comment]
> @JazzarKarim What's the game plan for new PRs on main branch? Do they need to be duplicated for the legal name branch?

I think Argus mentioned that we'll be rebasing every week or every sprint so that the branch doesn't fall behind a lot again. @argush3 Please correct me if I'm wrong or missed anything.

File: src/interfaces/store-interfaces/state-interfaces/business-interface.ts
Comment (severinbeauvais) [reply src/interfaces/store-interfaces/state-interfaces/business-interface.ts:1]: Oh, my duh. Yes, BusinessIF is the same except for the new object.

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Pull request merged**